lnd.go

```diff
@@ -1187,8 +1193,13 @@ func createWalletUnlockerService(cfg *Config) *walletunlocker.UnlockerService {
 	macaroonFiles := []string{
 		cfg.AdminMacPath, cfg.ReadMacPath, cfg.InvoiceMacPath,
 	}
+
+	var remoteWalletDB kvdb.Backend
+	if cfg.Cluster.EnableLeaderElection {
+		remoteWalletDB = remoteDB
+	}
 	return walletunlocker.New(
-		chainConfig.ChainDir, cfg.ActiveNetParams.Params,
+		chainConfig.ChainDir, cfg.ActiveNetParams.Params, remoteWalletDB,
```

Comment:
Otherwise this receives a `nil` interface pointer?

Reply:
The remote walletdb backend is now passed separately after elected as leader, since we need to hook the unlocker service to the RPC server before starting to serve.